_dashboards/index.md

```diff
@@ -11,15 +11,56 @@ redirect_from:
 
-## Get started with OpenSearch Dashboards
+![Sample dashboards](../images/dashboards-example.png)
 
-1. After starting OpenSearch Dashboards, you can access it at port 5601. For example, http://localhost:5601.
-1. Log in with the default username `admin` and password `admin`.
-1. Choose **Try our sample data** and add the sample flight data.
-1. Choose **Discover** and search for a few flights.
-1. Choose **Dashboard**, **[Flights] Global Flight Dashboard**, and wait for the dashboard to load.
+# Try OpenSearch Dashboards
+You can try OpenSearch Dashboards' features in the [OpenSearch Dashboards Playground](https://playground.opensearch.org/app/home).
+
+Choose **OpenSearch Dashboards > Dashboards > Create dashboard** and explore the sample Global Flight, Web Traffic, and Revenue dashboards.
+
+![Dashboards playground screenshot](../images/dashboards-playground.png)
+
+# OpenSearch Dashboards for your use case
+
+OpenSearch Dashboards makes it easy to visualize data for a number of use cases such as log analytics, application search, enterprise search, and more. Here are a few examples.  
+
+<table>
+<thead>
+</thead>
+<tbody>
+<tr>
+<td>Visualize log and trace data with interactive log analytics<img src="{{site.url}}{{site.baseurl}}/images/trace_log_correlation-web.jpg" alt="Log analytics" width="400" style="float: left; margin-right: 15px;"/></td>
+<td>Centralize and analyze logs for real-time threat detection and incident management<img src="{{site.url}}{{site.baseurl}}//images/detection.png" alt="Anomaly detection" width="400" style="float: left; margin-right: 15px;"/></td>
+</tr>
+<tr>
+<td>Quickly find relvant data within your applications, websites, and data catalogs<img src="{{site.url}}{{site.baseurl}}/images/drag-and-drop.png" alt="Data overview" width="400" style="float: left; margin-right: 15px;"/></td>
+<td>Find and fix problems and improve application health at a glance<img src="{{site.url}}{{site.baseurl}}/images/dashboards5.PNG" alt="Event analysis" width="400" style="float: left; margin-right: 15px;"/></td>
+</tr>
+</tbody>
+</table>
+
+# Install and create OpenSearch Dashboards
+
+After installing OpenSearch, you can install OpenSearch Dashboards, with different installation methods to choose. For more information, see [OpenSearch Installation Instructions](https://opensearch.org/docs/latest/opensearch/install/index/).
+
+You can follow these steps to create a dashboard using sample data. 
+
+1. After installing OpenSearch Dashboards, open it at [http://localhost:5601](http://localhost:5601/app/home#/).
+2. Select **Add sample data**, and select **Add data** for the sample  flight data dataset. 
+3. From the top menu, choose **Discover** and search for a few flights.
+4. Choose **Dashboard**, **[Flights]** Global Flight Dashboard.
+
+Your dashboard will look different than the example below.
```

Comment:
Is there any reason it would look different? If all they did was add the flight dataset and pull up a pre-made Flights dashboard, it should look exactly the same. Or is the picture not reflecting what they should see?